src/tabmat/constructor.py

```diff
@@ -148,13 +171,25 @@ def from_pandas(
         warnings.warn(
             f"Columns {ignored_cols} were ignored. Make sure they have a valid dtype."
         )
-    if dense_columns:
-        matrices.append(_dense_matrix(df, dense_columns, dtype))
-        indices.append(dense_indices)
+    if dense_dfidx:
+        matrices.append(
+            _dense_matrix(
```

Comment:
Not sure it's worth keeping the `_dense_matrix` function. We do not get the same clarity gains as before. What do you think @lbittarello ?

Reply:
I don't think it's worth keeping either of them. 👍